### PR TITLE
Added docs with instructions on adding monitoring to services

### PR DESCRIPTION
**Pull request by HumairAK**
Related [Issue](https://github.com/operate-first/apps/issues/108). These docs describe how users can use the ODH prometheus to monitor their services.

### CODE REVIEW

**Review by tumido — COMMENTED** [review]
/lgtm

File: docs/add_service_monitoring.md
Comment (tumido) [inline docs/add_service_monitoring.md:23]: Is the operate-first really needed in here? I think the `monitoring-rbac` should be enough

Comment (HumairAK) [reply docs/add_service_monitoring.md]: good call, fixed

File: docs/add_service_monitoring.md
Comment (hemajv) [inline docs/add_service_monitoring.md:7]: `group all your services into one service monitors` -> `group all your services into one service monitor` ?

**Review by hemajv — COMMENTED** [review]
thanks for the quick write up! :smile: 
/lgtm

File: docs/add_service_monitoring.md
Comment (HumairAK) [reply docs/add_service_monitoring.md]: fixed!

**Comment by anishasthana** [comment]
Can we get a review from one of our users? (i.e someone like @harshad16 )

**Review by harshad16 — COMMENTED** [review]
/lgtm
Thanks for the addition.
Should we mention somewhere that it looks for `/metrics` endpoint as default(not entirely sure of this), if not mentioned any other in service monitor?

**Review by tumido — COMMENTED** [review]
/lgtm

I'll approve and merge once @harshad16's question is addressed.

**Comment by HumairAK** [comment]
>  Should we mention somewhere that it looks for /metrics endpoint as default(not entirely sure of this), if not mentioned in service monitor? 
@harshad16 
Is it the default? I'm not sure, I don't see anything mentioned in the [docs here](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint)

**Comment by harshad16** [comment]
> Is it the default? I'm not sure, I don't see anything mentioned in the [docs here](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint) 
i dont see it either, i m confusing this with the target in prometheus job.
ignore my previous question.
/lgtm
this is good to be merged.

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**
